### PR TITLE
Merge 5.0.3xx to 6.0.1xx preview4

### PR DESCRIPTION
**Pull request by dsplaisted**
Current issues with merge:

- NETSDKXXXX message codes overlapped
- Blazor tests are failing with self-contained Exe reference errors

### CODE REVIEW

File: NuGet.config.orig [hunk 1]
```diff
@@ -0,0 +1,43 @@
+<?xml version="1.0" encoding="utf-8"?>
```
Comment (sfoslund) [inline NuGet.config.orig:1]: orig files got checked in here

**Review by sfoslund — CHANGES_REQUESTED** [review]

File: NuGet.config.orig (hunk 1 above)
Comment (dsplaisted) [reply NuGet.config.orig]: @sfoslund  This should be fixed, could you re-review?

File: sdk.sln [hunk 2]
```diff
@@ -160,31 +160,31 @@ Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Microsoft.NET.Sdk.Worker.Ta
 EndProject
 Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Microsoft.NET.Sdk.Publish.Tasks.Tests", "src\Tests\Microsoft.NET.Sdk.Publish.Tasks.Tests\Microsoft.NET.Sdk.Publish.Tasks.Tests.csproj", "{1F26FAE6-2654-4E83-8936-146134638C59}"
 EndProject
-Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "Sdk", "Sdk", "{F717D467-EF88-49B9-AD67-55FC6350A9DE}"
+Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "Sdk (2)", "Sdk (2)", "{F717D467-EF88-49B9-AD67-55FC6350A9DE}"
```
Comment (sfoslund) [inline sdk.sln:163]: I don't see these `(2)`'s in the 5.0.3xx branch, where are they coming from?

**Review by sfoslund — APPROVED** [review]

File: sdk.sln (hunk 2 above)
Comment (dsplaisted) [reply sdk.sln:163]: I added another Targets folder.  It seems like VS doesn't allow solution folders with the same name even if they're in different parts of the tree, so it ends up appending (2) to existing folders.

**Pull request merged**